### PR TITLE
GH-287 OpenMaps Implementation (#361)

### PR DESCRIPTION
**Pull request by jamesmontemagno**
### Description of Change ###

Describe your changes here. 

### Bugs Fixed ###

- Related to issue #361 

Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR.

### API Changes ###

List all API changes here (or just put None), example:

Added: 
 
- string - string Class.Property { get; set; } 
- void Class.Method();

Changed:

 - object Cell.OldPropertyName => object Cell.NewPropertyName

### Behavioral Changes ###

Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase.

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.Shared/LaunchMaps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/LaunchMaps_Tests.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/HomeViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/HomeViewModel.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/LaunchMapsViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/LaunchMapsViewModel.cs) | :white_check_mark:Succeeded | |
[Tests/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Tests/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/LaunchMaps.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/LaunchMaps.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/LaunchMaps.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/LaunchMaps.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/LaunchMaps.netstandard.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/LaunchMaps.netstandard.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/LaunchMaps.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/LaunchMaps.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/LaunchMaps.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/LaunchMaps.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapNavigation.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapNavigation.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/LaunchMaps.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/LaunchMaps.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapLaunchOptions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapLaunchOptions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapNavigation.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapNavigation.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Placemark.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Placemark.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/TextToSpeech.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/TextToSpeech.xml) | :white_check_mark:Succeeded | |
[docs/en/index.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/index.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807232021343067-404%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=STmy3gPBGiGTi%2F85K1GX0T0iMUgEpSYzmg%2FbAh0VaN0%3D&st=2018-07-23T20%3A17%3A46Z&se=2018-08-23T20%3A22%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.Shared/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/HomeViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/HomeViewModel.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/MapsViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/MapsViewModel.cs) | :white_check_mark:Succeeded | |
[Tests/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Tests/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapDirectionsMode.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapDirectionsMode.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.netstandard.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.netstandard.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/SecureStorage/SecureStorage.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/SecureStorage/SecureStorage.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/LocationExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/LocationExtensions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/mdoc.targets](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/mdoc.targets) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/LocationExtensions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/LocationExtensions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapDirectionsMode.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapDirectionsMode.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Maps.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Maps.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapsLaunchOptions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapsLaunchOptions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Placemark.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Placemark.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/PlacemarkExtensions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/PlacemarkExtensions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/TextToSpeech.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/TextToSpeech.xml) | :white_check_mark:Succeeded | |
[docs/en/index.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/index.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C24%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807241532242877-404%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=l%2FWmgJ5dLn4Xup2uYkVDm6RCMrjtoqcSKT3CM%2B2stlc%3D&st=2018-07-24T15%3A28%3A32Z&se=2018-08-24T15%3A33%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Xamarin.Essentials/Maps/Maps.shared.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            if (location == null)
+                throw new ArgumentNullException(nameof(location));
+
+            if (options == null)
+                throw new ArgumentNullException(nameof(options));
+
+            return PlatformOpenMapsAsync(location.Latitude, location.Longitude, options);
+        }
+
+        public static Task OpenAsync(double latitude, double longitude, MapsLaunchOptions options)
```
Comment (Redth) [inline Xamarin.Essentials/Maps/Maps.shared.cs]: Can we have an overload for `OpenAsync(double latitude, double longitude)` ?

**Comment by jamesmontemagno** [comment]
Done @Redth

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.Shared/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/HomeViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/HomeViewModel.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/MapsViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/MapsViewModel.cs) | :white_check_mark:Succeeded | |
[Tests/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Tests/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapDirectionsMode.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapDirectionsMode.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.netstandard.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.netstandard.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/SecureStorage/SecureStorage.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/SecureStorage/SecureStorage.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/LocationExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/LocationExtensions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/mdoc.targets](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/mdoc.targets) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/LocationExtensions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/LocationExtensions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapDirectionsMode.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapDirectionsMode.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Maps.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Maps.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapsLaunchOptions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapsLaunchOptions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Placemark.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Placemark.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/PlacemarkExtensions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/PlacemarkExtensions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/TextToSpeech.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/TextToSpeech.xml) | :white_check_mark:Succeeded | |
[docs/en/index.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/index.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C24%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807242154348923-404%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BM%2BGsWCLAq1Ri9cLPo6da6%2FgzI46FracQBwWp4AHnlM%3D&st=2018-07-24T21%3A50%3A43Z&se=2018-08-24T21%3A55%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/DeviceTests.Shared.projitems) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.Shared/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/DeviceTests/DeviceTests.Shared/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml) | :white_check_mark:Succeeded | |
[Samples/Samples/View/MapsPage.xaml.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/View/MapsPage.xaml.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/HomeViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/HomeViewModel.cs) | :white_check_mark:Succeeded | |
[Samples/Samples/ViewModel/MapsViewModel.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Samples/Samples/ViewModel/MapsViewModel.cs) | :white_check_mark:Succeeded | |
[Tests/Maps_Tests.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Tests/Maps_Tests.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapDirectionsMode.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapDirectionsMode.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/MapLaunchOptions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.netstandard.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.netstandard.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Maps/Maps.uwp.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Maps/Maps.uwp.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/SecureStorage/SecureStorage.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/SecureStorage/SecureStorage.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/LocationExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/LocationExtensions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.android.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.android.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.ios.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.ios.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Types/PlacemarkExtensions.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/mdoc.targets](https://github.com/xamarin/Essentials/blob/feature/open-maps/Xamarin.Essentials/mdoc.targets) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/LocationExtensions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/LocationExtensions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapDirectionsMode.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapDirectionsMode.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Maps.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Maps.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/MapsLaunchOptions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/MapsLaunchOptions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Placemark.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/Placemark.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/PlacemarkExtensions.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/PlacemarkExtensions.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/TextToSpeech.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/Xamarin.Essentials/TextToSpeech.xml) | :white_check_mark:Succeeded | |
[docs/en/index.xml](https://github.com/xamarin/Essentials/blob/feature/open-maps/docs/en/index.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C25%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201807251621047335-404%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2Bsz9B7hfvi9JQsiVEYm0JRUyoXjqybpWAsr9kwPo3E8%3D&st=2018-07-25T16%3A17%3A20Z&se=2018-08-25T16%3A22%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**